### ISSUE TITLE
Installer error

### ISSUE THREAD

**Issue by craigh** [zikula/core#4083]
I'm not sure if this is an error. but it is new since changes to config directory. @Guite can you look at it please?

```shell
[2020-01-25T15:02:46.300183+00:00] php.DEBUG: Warning: chmod(): No such file or directory {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/Applications/MAMP/htdocs/core.git/src/Zikula/CoreInstallerBundle/Helper/ParameterHelper.php","line":180,"trace":[{"file":"/Applications/MAMP/htdocs/core.git/src/Zikula/CoreInstallerBundle/Helper/StageHelper.php","line":163,"function":"protectFiles","class":"Zikula\\Bundle\\CoreInstallerBundle\\Helper\\ParameterHelper","type":"->"}],"count":1}}} []

[2020-01-25T15:02:46.343148+00:00] php.DEBUG: Warning: chmod(): No such file or directory {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/Applications/MAMP/htdocs/core.git/src/Zikula/CoreInstallerBundle/Helper/ParameterHelper.php","line":182,"trace":[{"file":"/Applications/MAMP/htdocs/core.git/src/Zikula/CoreInstallerBundle/Helper/StageHelper.php","line":163,"function":"protectFiles","class":"Zikula\\Bundle\\CoreInstallerBundle\\Helper\\ParameterHelper","type":"->"}],"count":1}}} []

[2020-01-25T15:02:46.343795+00:00] php.DEBUG: Warning: chmod(): No such file or directory {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/Applications/MAMP/htdocs/core.git/src/Zikula/CoreInstallerBundle/Helper/ParameterHelper.php","line":184,"trace":[{"file":"/Applications/MAMP/htdocs/core.git/src/Zikula/CoreInstallerBundle/Helper/StageHelper.php","line":163,"function":"protectFiles","class":"Zikula\\Bundle\\CoreInstallerBundle\\Helper\\ParameterHelper","type":"->"}],"count":1}}} []
```

**Comment by Guite** [comment]
Can't replicate here. Need to reinvestigate later I think. Maybe also environment-related ...

**Comment by craigh** [comment]
didn't reproduce

**Comment by craigh** [comment]
reproduced again

```shell
 Protect configuration files
 Protecting configuration files
13:53:13 DEBUG     [php] Warning: chmod(): No such file or directory
[
  "exception" => Symfony\Component\ErrorHandler\Exception\SilencedErrorContext^ {
    +count: 1
    -severity: E_WARNING
    trace: {
      ./src/Zikula/CoreInstallerBundle/Helper/ParameterHelper.php:180 {
        Zikula\Bundle\CoreInstallerBundle\Helper\ParameterHelper->protectFiles(): bool^
        › foreach ($files as $file) {
        ›     @chmod($file, 0400);
        ›     if (!is_readable($file)) {
      }
      ./src/Zikula/CoreInstallerBundle/Helper/StageHelper.php:163 { …}
    }
  }
]
13:53:13 DEBUG     [php] Warning: chmod(): No such file or directory
[
  "exception" => Symfony\Component\ErrorHandler\Exception\SilencedErrorContext^ {
    +count: 1
    -severity: E_WARNING
    trace: {
      ./src/Zikula/CoreInstallerBundle/Helper/ParameterHelper.php:182 {
        Zikula\Bundle\CoreInstallerBundle\Helper\ParameterHelper->protectFiles(): bool^
        › if (!is_readable($file)) {
        ›     @chmod($file, 0440);
        ›     if (!is_readable($file)) {
      }
      ./src/Zikula/CoreInstallerBundle/Helper/StageHelper.php:163 { …}
    }
  }
]
13:53:13 DEBUG     [php] Warning: chmod(): No such file or directory
[
  "exception" => Symfony\Component\ErrorHandler\Exception\SilencedErrorContext^ {
    +count: 1
    -severity: E_WARNING
    trace: {
      ./src/Zikula/CoreInstallerBundle/Helper/ParameterHelper.php:184 {
        Zikula\Bundle\CoreInstallerBundle\Helper\ParameterHelper->protectFiles(): bool^
        › if (!is_readable($file)) {
        ›     @chmod($file, 0444);
        › }
      }
      ./src/Zikula/CoreInstallerBundle/Helper/StageHelper.php:163 { …}
    }
  }
]
```

**Comment by Guite** [comment]
CLI and/or web installer?

**Comment by craigh** [comment]
CLI

**Comment by craigh** [comment]
probably the same issue we've been discussing CLI v http

**Comment by Guite** [comment]
@craigh This should be fixed now. Could you please review?

**Comment by Guite** [comment]
Fixed in #4096
